3D/Web/Findings.txt

```diff
@@ -0,0 +1,12 @@
+0 0 0
+0 0 0
+1 1 1
+#ff11ff
+0 -2 0
+45 45 0
```

Comment:
Sa te uiti la terminatia din fisier a linilor sa se termine in "\n" in loc de "\r\n"